src/main/res/values/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <!-- Copyright (C) 2015 Cagri Cetin (cagricetin@mail.usf.edu), University of South Florida
+    <!-- Copyright (C) 2015 University of South Florida (cagricetin@mail.usf.edu)
     and individual contributors.
```

Comment:
Let's remove `and individual contributors.` and replace them with the known contributors, if we know this.  I started including this wording in OBA Android, but in retrospect it was a mistake.  We should try to list all known authors.